assignment/Assignment2.kt

```diff
@@ -0,0 +1,50 @@
+package assignment
+
+import java.util.Scanner
+
+class KStack<T>{
+    private val stack = mutableListOf<T>()
+
+    fun push(element: T){
+        stack.add(element)
+    }
+    fun pop(): T?{
+        if(stack.size == 0)
+            return null
+        return stack.removeLast()
+    }
+    fun size(): Int{
+        return stack.size;
+    }
```

Comment:
```suggestion
    fun size() = stack.size
```

리턴할 때 한줄이다 싶으면은 인라인으로 위와같이 적을 수 있습니다.